### ISSUE TITLE
Plugin not working against native mobile android application

### ISSUE THREAD

**Issue by naddison** [wswebcreation/webdriver-image-comparison#19]
**Environment (please complete the following information):**
 - Node.js version: 8.12
 - NPM version: 6.4.1

Capabilities
```
 { automationName: 'UiAutomator2',
        Application: 'kitchensinkapp',
        deviceName: 'Android Emulator',
        avd: 'Pixel_2_API_28',
        platformVersion: '9.0',
        platformName: 'ANDROID',
        appPackage: 'com.kitchensinkapp',
        appActivity: 'com.kitchensinkapp.MainActivity' 
}
```

**Config of the automation framework + plugin**
This was in use in a WDIO test using the comparison plugin.

**Use case**
We call `saveFullPageScreen()`

```
await browser.saveFullPageScreen('fullPage', { /* some options*/ });
```

which results in this error.

```
2019-06-06T21:52:46.068Z INFO webdriver: COMMAND saveFullPageScreen("fullPage", <object>)
2019-06-06T21:52:46.074Z INFO webdriver: COMMAND executeScript("return (function hideScrollBars(hide) {
    if (hide) {
        document.body.style.overflow = 'hidden';
    }
    else {
        document.body.style.overflow = '';
    }
}).apply(null, arguments)", <object>)
2019-06-06T21:52:46.074Z INFO webdriver: [POST] http://10.10.29.235:4000/wd/hub/session/fafbde0f-f066-41b9-89c8-67806a13f9ff/execute/sync
2019-06-06T21:52:46.074Z INFO webdriver: DATA { script:
   'return (function hideScrollBars(hide) {\n    if (hide) {\n        document.body.style.overflow = \'hidden\';\n    }\n    else {\n        document.body.style.overflow = \'\';\n    }\n}).apply(null, arguments)',
  args: [ false ] }
2019-06-06T21:52:46.079Z DEBUG webdriver: request failed due to response error: unknown error
2019-06-06T21:52:46.080Z WARN webdriver: Request failed due to Method is not implemented
2019-06-06T21:52:46.080Z INFO webdriver: Retrying 1/3
```

1. `saveFullPageScreen()` comes from https://github.com/wswebcreation/wdio-image-comparison-service/blob/master/lib/index.js#L65
2. Which in turn calls `saveFullPageScreen()` from https://github.com/wswebcreation/webdriver-image-comparison/blob/master/lib/commands/saveFullPageScreen.ts#L16
3. This error specifically is thrown from `beforeScreenshot()` which is in https://github.com/wswebcreation/webdriver-image-comparison/blob/master/lib/helpers/beforeScreenshot.ts#L13

The capabilities:
```
2019-06-06T21:52:34.420Z INFO webdriver: DATA { capabilities:
   { alwaysMatch:
      { automationName: 'UiAutomator2',
        Application: 'kitchensinkapp',
        deviceName: 'Android Emulator',
        avd: 'Pixel_2_API_28',
        platformVersion: '9.0',
        platformName: 'ANDROID',
        appPackage: 'com.kitchensinkapp',
        appActivity: 'com.kitchensinkapp.MainActivity' },
     firstMatch: [ {} ] },
  desiredCapabilities:
   { automationName: 'UiAutomator2',
     Application: 'kitchensinkapp',
     deviceName: 'Android Emulator',
     avd: 'Pixel_2_API_28',
     platformVersion: '9.0',
     platformName: 'ANDROID',
     appPackage: 'com.kitchensinkapp',
     appActivity: 'com.kitchensinkapp.MainActivity' } }
```

Code calling the saveFullPageScreen()
```
describe('Do the thing', () => {
    it('should do the thing', async () => {
        
		... do some button clicks...
		
        await browser.saveFullPageScreen('fullPage', { /* some options*/ });

    });
});
```

Basically my question is does this plugin support mobile native apps? I thought using browser.execute() from WDIO would ensure the command would get mapped to Appium protocol. 

It is news to me that Appium apparently does not implement the `:sessionId/execute/sync` endpoint?

**Comment by wswebcreation** [comment]
Hi @naddison 

This module is only for web, see also the readme

> wdio-image-comparison-service is a lightweight WebdriverIO service for browsers / mobile browsers / hybrid apps to do image comparison on screens, elements or full page screens.

I also have a module for native apps, that's this module https://github.com/wswebcreation/wdio-native-app-compare

Keep in mind that you don't have the option to create a fullpage screenshot because a native app behaves differently.

Going to close it now because it's not an issue for this module
